appdev/apparmor.rst

```diff
@@ -0,0 +1,295 @@
+    Usage: How common it is to use this policy (from apparmor file)
+
+    Optional longer description
+
+Policy usage affects whether your app will be accepted by the OpenStore. Apps containing policies with common usage are generally accepted immediately, while reserved usage policies will need to be manually reviewed.
+
+accounts
+--------
+
+Description: Can use Online Accounts.
+
+Usage: common
+
+The accounts policy gives your app the permissions it needs to access the `Online Accounts API`_.
+
+
+audio
+-----
+
+Description: Can play audio (allows playing remote content via media-hub)
+
+Usage: common
+
+The audio policy is needed for your app to play audio via pulseaudio or media-hub. The permission also gives it the ability to send album art to the thumbnailer service, which is then shown on the sound indicator.
+
+
+bluetooth
+---------
+
+Description: Use bluetooth (bluez5) as an administrator.
+
+Usage: reserved
+
+This policy grants unrestricted access to Bluetooth devices. It is provided for administration of bluetooth and as a stepping stone towards developing a safe bluetooth API all apps can access.
+
+
+calendar
+--------
+
+Description: Can access the calendar.
+
+Usage: reserved
+
+Calendar grants access to the Evolution dataserver's calendar and alarms APIs. It also grants access to sync-monitor.
+
+This policy is reserved since it grants free access to all calendars on the device at any time. The legacy bug about this situation is `LP #1227824`_ .
+
+
+camera
+------
+
+Description: Can access the camera(s)
+
+Usage: common
+
+The camera policy grants access to device cameras.
+
+
+connectivity
+------------
+
+Description: Can access coarse network connectivity information
+
+Usage: common
+
+The connectivity policy allows apps to determine rough information about the device's connectivity. This includes whether the device is connected to the Internet and whether it is connected via a Wi-Fi or mobile data connection.
+
+
+contacts
+--------
+
+Description: Can access contacts.
+
+Usage: reserved
+
+The contacts policy allows apps to access the device user's contacts list. It is marked as reserved because it allows access to sync-monitor and unfettered access to the address book.
+
+
+content_exchange
+----------------
+
+Description: Can request/import data from other applications
+
+Usage: common
+
+Using the content_exchange policy allows your app to be a consumer of content on content-hub.
+
+
+content_exchange_source
+-----------------------
+
+Description: Can provide/export data to other applications
+
+Usage: common
+
+The content_exchange_source policy allows your app to provide content on content-hub.
+
+
+debug
+-----
+
+Description: Use special debugging tools. This should only be used in development and not for production packages. Note: use of this policy group rovides significantly different confinement than normal and is not considered secure. You should never run untrusted programs using this policy group.
```

Comment:
Typo provides

Do you know what a developer could get from this? Is it useful during developing testing? How would a developer use it?